internal/mill/exec.go

```diff
@@ -7,17 +7,61 @@ import (
 	"path/filepath"
 
 	"github.com/google/skylark"
+	"github.com/windmilleng/pets/internal/proc"
 )
 
+type Probe struct {
```

Comment:
we should probably come up with a better name for this. 'Interpreter'? 'Execer'? or if we want a good pun, 'PetSitter'?